src/Execution/DataLoader/ModelRelationFetcher.php

```diff
@@ -419,4 +419,19 @@ protected function getRelationInstance(string $relationName): Relation
             ->newModelQuery()
             ->getRelation($relationName);
     }
+
+    /**
+     * Get model key. Support composite primary keys.
+     * Ex: $primaryKey = ['key1', 'key2'];.
+     *
+     * @param Model $model
+     *
+     * @return string
+     */
+    protected function getModelKey(Model $model)
```

Comment:
This function is kind of duplicated from BatchLoader, can we keep this DRY?

Reply:
What do you suggest? Don’t where to place it.

Reply:
@spawnia done!